epub33/core/index.html

```diff
@@ -1014,9 +1014,20 @@ <h3>XML Conformance</h3>
 								Documents</a> [[!XML-NAMES]].</p>
 					</li>
 					<li>
-						<p id="confreq-xml-extmarkupdecl">
-							<a href="https://www.w3.org/TR/2008/REC-xml-20081126/#NT-ExternalID">External
-								identifiers</a> MUST NOT appear in the document type declaration [[!XML]].</p>
+						<p id="confreq-xml-identifiers">It MAY only include a <a
+								href="https://www.w3.org/TR/2008/REC-xml-20081126/#dt-doctype">document type
+								declaration</a> that references the <a
+								href="https://www.w3.org/TR/2008/REC-xml-20081126/#dt-pubid">public identifier</a> and
+								<a href="https://www.w3.org/TR/2008/REC-xml-20081126/#dt-sysid">system identifier</a>
+							for a format referenced in <a href="#app-identifiers-allowed"></a>, or that omits <a
+								href="https://www.w3.org/TR/2008/REC-xml-20081126/#NT-ExternalID">external
+								identifiers</a> [[!XML]].</p>
+					</li>
+					<li>
+						<p id="confreq-xml-entities">It MUST NOT declare external <a
+								href="https://www.w3.org/TR/2008/REC-xml-20081126/#dt-parsedent">parsed entities</a> or
+							external <a href="https://www.w3.org/TR/2008/REC-xml-20081126/#dt-PE">parameter entities</a>
+							in an internal DTD subset [[!XML]].</p>
```

Comment:
@rdeltour 

The taxonomy of entities in XML is extremely difficult to understand.  I believe that I read every sentence in the first edition of XML at least 20 times.  But this taxonomy is still confusing to me.

You correctly pointed out that parameter entities are parsed entities referenced within DTDs.  Stand corrected.  But general entities include unparsed entities as well.  So, "parsed entities" are not broader than "general entities".

IMHO, internal DTD subsets must not declare

1. external parameter entities
2. external general parsed entities
3. unparsed entities (which are always external and general)

The sum of the first three is "external entities".  We can simply say that declarations of external entities are disallowed in internal DTD subsets.  But this is just too difficult for everybody.

The original wording in EPUB 3 is quite good since it is free from the confusing taxonomy.

We might want to slightly change the original wording.  How about this?

- External identifiers MUST NOT appear in the internal DTD subset [XML].

Reply:
Is there terminology that could cover "all things external" so we don't have to get into a list of technical terms?

For example:

> "It MUST NOT reference external resources from the internal DTD subset [[!XML]].

Reply:
Ah, we were writing at the same time! I'm fine with any wording that encompasses all we need to restrict without being overly detailed.